### ISSUE TITLE
Select a role to specific players in order to equilibrate the game 

### ISSUE THREAD

**Issue by santicarmo31** [santicarmo31/Mafia#29]
If I know that someone is very good playing as specific ROLE heh should have the feature to set that specific role in order to make the game more equilibrate and interesting, also imagine that someone is new the best option form them would be civilians at the first game, to look at how the game is.

**Comment by lramirezsuarez** [comment]
Maybe in the detail view controller, add a role selector, and refresh the rest of the roles in the game view controller when going back.

**Comment by santicarmo31** [comment]
Yeah @lramirezsuarez I have thought the same, the best place would be the detail. I'm thinking in how can we then shuffle the rest of roles.

**Comment by lramirezsuarez** [comment]
Put a flag in the entity when asigning manually the role, then filter the players in the presenter when that flag is assigned and shuffle the rest of the roles accordingly to that flag.

**Comment by santicarmo31** [comment]
Would you like to do it ?

**Comment by lramirezsuarez** [comment]
A nice challenge, sure!
